apps/pokedex/pokedex.star

```diff
@@ -9,9 +9,13 @@ load("http.star", "http")
 load("render.star", "render")
 load("schema.star", "schema")
 load("time.star", "time")
+load("encoding/base64.star", "base64")
```

Comment:
Nit: should these be sorted?

Reply:
If the linter passes, then it's good to go! But maybe the linter should require these to be sorted 😅 . Will fix.